src/main/scala/beam/agentsim/agents/parking/ChoosesParking.scala

```diff
@@ -182,6 +176,10 @@ trait ChoosesParking extends {
         responses: (RoutingResponse, RoutingResponse),
         data @ BasePersonData(_, _, _, _, _, _, _, _, _)
         ) =>
+      val (r1, r2) = responses
```

Comment:
Similar to ChoosesMode, ChoosesParking is mostly a planning class and I don't think we want to throw movement events here.  It should all happen in DrivesVehicle at the same time PathTraversal is processed. This is the true point when we know a movement through links has been made.